### PR TITLE
Add apicurio-keycloak-extensions

### PR DESCRIPTION
**Pull request by ndmaxar**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by ndmaxar** [comment]
@oblute and @rluria14 ok to be maintainers?

**Comment by scopatz** [comment]
CC @mariusvniekerk - does this look right?

**Comment by mariusvniekerk** [comment]
Where does this place the resulting jar?

**Comment by ndmaxar** [comment]
> Where does this place the resulting jar?

In a subdirectory at `social/target/jarfile`.

**Comment by ndmaxar** [comment]
@mariusvniekerk is this ok or should I do something else with the jar?

**Comment by mariusvniekerk** [comment]
i'm pushing some stuff to make this fit a bit better,  how are you planning to use this jar, since it requires other stuff to be usable

**Comment by mariusvniekerk** [comment]
changed build to put the built jars in `$PREFIX/jars`

**Comment by scopatz** [comment]
LGTM

**Comment by mariusvniekerk** [comment]
@scopatz, this can probably be noarch: generic?

**Comment by scopatz** [comment]
Yeah that is worth a try

**Comment by ndmaxar** [comment]
Added

File: recipes/apicurio-keycloak-extensions/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+build:
+  number: 0
+  noarch: generic
+  script:
+    - set -x
+    - mvn package
+    - mkdir -p $PREFIX/jars                                                                             # [unix]
+    - mkdir "%PREFIX%\jars"                                                                             # [win]
+    - cp social/target/apicurio-keycloak-extensions-social-{{ version }}.Final.jar $PREFIX/jars         # [unix]
+    - copy "social\target\apicurio-keycloak-extensions-social-{{ version }}.Final.jar" "%PREFIX%\jars"  # [win]
```
Comment (scopatz) [inline recipes/apicurio-keycloak-extensions/meta.yaml:21]: Please just make a build.sh file. As noarch generic it doesn't need to build on win

Comment (ndmaxar) [reply recipes/apicurio-keycloak-extensions/meta.yaml:21]: Can do

Comment (ndmaxar) [reply recipes/apicurio-keycloak-extensions/meta.yaml]: Do I need to add a skip for win?

Comment (mariusvniekerk) [reply recipes/apicurio-keycloak-extensions/meta.yaml]: nah, you don't need to skip win

**Comment by ndmaxar** [comment]
Got it ready to go

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**